postprocessing/janus-pp-rec.c

```diff
@@ -949,8 +962,9 @@ int main(int argc, char *argv[])
 			} else {
 				rtp_header_len += rtp_read_n;
 			}
-			if(audio_level_extmap_id > 0)
+			if(audio_level_extmap_id > 0) {
 				janus_pp_rtp_header_extension_parse_audio_level(prebuffer, len, audio_level_extmap_id, &audiolevel);
+			}
```

Comment:
I don't understand why you added these brackets here. Please only make changes relevant to your contribution.